src/index.js

```diff
@@ -1,14 +1,18 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const todolist = [{ description: 'going to the gym' }, { description: 'take a shower' }, { description: 'have breakfast' }];
 
-  // Lodash, now imported by this script
-
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-
-  return element;
+function iteration() {
+  const interactivelist = document.querySelector('.interactive-list');
+  for (let i = 0; i < todolist.length; i += 1) {
+    const listitem = `
+    <li class="listitem">
+    <input type="checkbox" class="checkbox" id="checkbox" name="listtodo">
+    <label id="description" for="todos">${todolist[i].description}</label>
+    <span class="material-symbols-outlined" id="more">more_vert</span></li>
+    `;
+    interactivelist.innerHTML += listitem;
+  }
 }
 
-document.body.appendChild(component());
\ No newline at end of file
+iteration(todolist);
```

Comment:
- :clap: You have made a brilliant attempt here, @edgardsoza :vulcan_salute: Just a few more to watch out for in order to fully match the [Project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md):
- [ ] Kindly implement the minimum requirement of at least `three key/value` pairs of `string`, `boolean`, and `number` as index per task in your `todolist` array of objects.
- [ ] The requirement suggest dynamic population when an `onload` event occurs but you seem to omitt the `event`. Consider adding an `event listener` so that you would have met all the changes.
- [ ] [OPTIONAL] Am I right about the fact that the `.gitignore` is ignoring your `./dist` folder?

- I have attached a screenshot so that you can have a more graphical feel of the project requirement :point_down: 

![image](https://user-images.githubusercontent.com/99715640/222250757-3af4c0e5-0a01-49b4-a517-682cfca6642f.png)

Reply:
I'm really sorry I got this late. It was due to the network. I'm glad it's been approved. Well done, champ! I knew you could do it 😉🚀